inc/define.php

```diff
@@ -68,6 +68,8 @@
 define("UNLOCK", 128);
 define("READ_ASSIGNED", 256);
 define("UPDATE_ASSIGNED", 512);
+define("READ_OWNED", 1024);
```

Comment:
I don't have the answer, but I hope using next bit will not conflict with some stored rights in DB.

Reply:
As far as I see, this value is not used in any assets or assignable items.